www/src/pages/pl/why.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Dlaczego CT3A?
+description: Dlaczego powinieneś wybrać Create T3 App dla swojego następnego projektu
+layout: ../../layouts/docs.astro
+lang: pl
+---
+
+Stworzyliśmy Create T3 App, ponieważ [Theo](https://twitter.com/t3dotgg) nie chciał przygotować szablonu składającego się ze swoich ulubionych technologii. Zainspirowany przez create-next-app, [CLI od Astro](https://astro.build) oraz ogólne zamiłowanie do typesafety, zespół Create T3 App ciężko pracował budując najlepszy możliwy punkt startowy dla nowych projektów korzystających z T3 Stacka.
```

Comment:
```suggestion
Stworzyliśmy Create T3 App, ponieważ [Theo](https://twitter.com/t3dotgg) nie chciał przygotować szablonu składającego się ze swoich ulubionych technologii. Zainspirowany przez create-next-app, [CLI od Astro](https://astro.build) oraz ogólne zamiłowanie do typesafety, zespół Create T3 App ciężko pracował budując najlepszy możliwy punkt startowy dla nowych projektów korzystających z T3 Stacku.
```
Isn't "stacku" more correct than "stacka"?

Reply:
@matibox Jak myślisz, robimy "stacku" czy "stacka"? Moim zdaniem oba pasują (nawet zależnie od zdania czasami jedno bardziej, czasami drugie :smile: ) a nie umiem znaleźć jakiejś zasady spolszczania takich rzeczy :thinking:

Reply:
Generalnie to ciężko takie rzeczy tłumaczyć, ale jak już to robimy to wszędzie tak samo. W tym zdaniu akurat stacka mi lepiej brzmi.

Reply:
Może zamiast T3 stacka/u to zamienimy te wyrazy i zrobimy stacka/u T3

`...zespół Create T3 App ciężko pracował budując najlepszy możliwy punkt startowy dla nowych projektów korzystających ze Stacka T3.`

Reply:
> Może zamiast T3 stacka/u to zamienimy te wyrazy i zrobimy stacka/u T3
> 
> `...zespół Create T3 App ciężko pracował budując najlepszy możliwy punkt startowy dla nowych projektów korzystających ze Stacka T3.`

W sumie dobry pomysł, wydaje mi się też że jak piszemy w tej kolejności to stacka by było z małej litery bo to raczej nie jest nazwa własna "T3 Stack" tylko po prostu określenie?

Reply:
> > Może zamiast T3 stacka/u to zamienimy te wyrazy i zrobimy stacka/u T3
> > `...zespół Create T3 App ciężko pracował budując najlepszy możliwy punkt startowy dla nowych projektów korzystających ze Stacka T3.`
> 
> W sumie dobry pomysł, wydaje mi się też że jak piszemy w tej kolejności to stacka by było z małej litery bo to raczej nie jest nazwa własna "T3 Stack" tylko po prostu określenie?

Jasne 👍